R/init.R

```diff
@@ -133,3 +141,25 @@ create_e2e_tests_structure <- function() {
   copy_template("e2e_tests")
   cli::cli_alert_success("E2E tests structure created.")
 }
+
+is_dir_home <- function(dir, force) {
+  if (!fs::dir_exists(dir) || force) {
+    return(FALSE)
+  }
+
+  home_path <- normalizePath("~")
```

Comment:
What does it return on Windows? It would be good to add a comment here explaining what is considered a home directory on Unix vs Windows.